### DISCUSSION TITLE
Upload 组件，选择好图片上传后，提交的值是 fileList数组，有没有好的办法变成 url 串的形式？

### DISCUSSION THREAD

**Discussion by lg970044** [alibaba/formily#2970] [Q&A 2.x] [answered]
应用 Upload 组件时，选择好图片上传后，后端收到的是 fileList数组，有没有好的办法，提交到后端的值 是 后端 想要的格式。比如 
图片的 url 串？

难道 只能在  onSubmit 去处理？

**Comment by janryWang** [comment, marked as answer]
二次封装Upload，拦截value/onChange，做一层转换
